src/utils/info.js

```diff
@@ -6,3 +6,10 @@ export const showToast = (message: string) => Toast.show(message, Toast.SHORT);
 
 export const showErrorAlert = (message: string, title: string) =>
   Alert.alert(title, message, [{ text: 'OK', onPress: () => {} }], { cancelable: true });
+
+export const showErrorFromException = (error: Error, title: string) =>
+  error
+  && error.data
+  && error.data.msg
+  // $FlowFixMe
+  && Alert.alert(title, error.data.msg, [{ text: 'OK', onPress: () => {} }], { cancelable: true });
```

Comment:
You can do the `error && error.data && error.data.msg` as a second parameter.
Not only will this be more simple, will not make Flow complain (Flow is correct to complain!) but it will also show the error with the `title` text regardless if the error object is defined.